core/templates/services/playthrough.service.ts

```diff
@@ -50,12 +49,11 @@ interface CycleIdentifier {
 })
 export class PlaythroughService {
   constructor(
-    private http: HttpClient,
     private explorationFeaturesService: ExplorationFeaturesService,
```

Comment:
Please have these indented by 4 spaces instead of 2.

Reply:
done